recipes/cli11/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(CLI11 REQUIRED CONFIG)
+
+add_executable(${CMAKE_PROJECT_NAME} ../test_package/test_package.cpp)
+set_property(TARGET ${CMAKE_PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+target_link_libraries(${CMAKE_PROJECT_NAME} CLI11::CLI11)
```

Comment:
this is important to note moving forward, but not a blocker for me

Reply:
Oh, I was just copy-pasting the old test_package to test_v1_package, as I figured it worked before with v1, so it would be important to test the new recipe worked with the old test.

I'll do that in future too.